### ISSUE TITLE
Follow up on congenital fibrosarcoma conversation

### ISSUE THREAD

**Issue by ckesserwan** [thePCCAS/onset_ages#3]
@jsaliba10
<img width="1762" alt="Screen Shot 2021-07-07 at 3 06 18 PM" src="https://user-images.githubusercontent.com/62956984/124815249-0da35200-df35-11eb-9cf2-069cea5d6113.png">
   It just came to my attention that in The WHO blue book, the preferred term is "Infantile fibrosarcoma" . Congenital fibrosarcoma is an acceptable related terminology. I do not think that this would affect your age categorization option.

**Comment by jsaliba10** [comment]
@ckesserwan You are correct. I forgot to mention during the discussion that the display name for the disease is "Congenital FS" based on what is in the disease ontology (DO). Infantile FS is the ICDO term and a synonym for congenital FS in the DO. 


[Link to EID8878](https://civicdb.org/events/genes/3983/summary/variants/3223/summary/evidence/8878/summary#evidence)
The following is from the paper:
"At 13 d of life, the patient underwent biopsy of the lesion at an outside hospital, which revealed a low-grade spindle cell proliferation with fascicular arrangement and myxoid background infiltrating fatty tissue resembling fibromatosis.  The lesion was classified as fibromatosis with the differential diagnosis including lipofibromatosis, although the possibility of the biopsy representing the edge of an IFS could not be excluded. Fluorescence in situ hybridization (FISH) for sarcoma translocations, including t(12;15)(p13;q25), was performed and was negative."

"At age 2, the left forearm mass began to enlarge, and the patient underwent repeat resection of >95% of the mass. Pathology from this sample (Fig. 1C) showed a more cellular spindle cell proliferation arranged in fascicles and sheets with hemangiopericytoma-like vascular channels with focal necrosis and increased mitotic figures. The differential diagnosis at this stage included fibrosarcoma, spindle cell embryonal rhabdomyosarcoma, leiomyosarcoma, malignant peripheral nerve sheath tumor, and synovial sarcoma. Overall, the pathology was characterized as high-grade spindle cell sarcoma, Pediatric Oncology Group (POG) grade 3/3, with a negative metastatic workup."

IFS doesn't completely fit into the age of onset ranges as It can be present at birth or can develop during the first 5 years, particularly in infants and toddlers younger than 2 years; 80 % of cases are diagnosed during the first year of life.

The mass was there at birth, but was not yet considered IFS. At the second biopsy at 2yr it was classified as IFS. This has me leaning onset should be based on the diagnosis which was set at the age of 2 and therefore childhood onset (1-5yr) and not congenital (birth) or infantile (28d to 1yr).  

Either way this will be tagged with a pediatric as pediatric evidence, but I am curious which onset term you think is best.

**Comment by ckesserwan** [comment]
@jsaliba10 They just missed the diagnosis of IFS on the first biopsy because they were looking at the conventional marker of IFS (ETV6/NTRK3). In this specific patient, the onset is congenital. In other cases, you may not be able to determine whether the onset was congenital, infantile or even of childhood onset
